src/__tests__/useFullscreen.spec.tsx

```diff
@@ -1,12 +1,26 @@
-import { renderHook } from '@testing-library/react-hooks';
-import { useFullscreen } from '../hooks/useFullscreen';
+import { renderHook, act } from "@testing-library/react-hooks";
+import { useFullscreen } from "../hooks/useFullscreen";
 
-describe('useFullscreen', () => {
-  it('should be defined', () => {
+describe("useFullscreen", () => {
+  it("should be defined", () => {
     expect(useFullscreen).toBeDefined();
   });
-  it('should return an object', () => {
+  it("should return an object", () => {
     const { result } = renderHook(() => useFullscreen());
-    expect(typeof result.current).toBe('object');
+    expect(typeof result.current).toBe("object");
+  });
+  it.only("should forward requestFullscreenOptions to requestFullscreen", () => {
+    (document as any).exitFullscreen = jest.fn();
```

Comment:
Do we need to assert `any` here? 🤔 Curious as to why this is needed.

Reply:
Previous, I put `globals as any`, but forgot to remove any when I changed to `document`

Reply:
I removed `any`